src/components/Header/Header.module.css

```diff
@@ -0,0 +1,28 @@
+.headerTitle {
+  justify-content: center;
+  font-family: Exo 2;
+  font-style: normal;
+  font-weight: 900;
+  font-size: 42px;
+  line-height: 50px;
+  display: flex;
+  margin-top: 50px;
+  color: #000000;
+}
+
+.headerDescription {
+  justify-content: center;
+  font-family: Exo 2;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 27px;
+  margin-top: 40px;
+  display: flex;
+  color: #000000;
+}
+
+.headerContainer {
+  background-image: url(./header.png);
```

Comment:
Add `background-repeat` and `background-size` properties to this: 
![image](https://user-images.githubusercontent.com/13375042/136849802-521d95b7-43b8-4a1b-9033-b13117fae7f8.png)

Right now the image is repeating and has a clear line break down the middle